docs/source/install/install_local.rst

```diff
@@ -33,9 +33,9 @@ Python. If Anaconda is not your default Python distribution, download and instal
            pip install fbpic[picmi]
 
    .. note::
-       Instead of using ``pip``, you can also install FBPIC from the souces,
+       Instead of using ``pip``, you can also install FBPIC from the sources,
        by cloning the `code from Github <https://github.com/fbpic/fbpic>`_,
-       and typing ``python setup.py install``.
+       and executing ``python3 -m pip install .`` from the main directory.
```

Comment:
```suggestion
       and executing ``python3 -m pip install .`` from the main directory.
       A shortcut for this is: ``python3 -m pip install git+https://github.com/fbpic/fbpic.git``
```

Reply:
Could not commit suggestion directly from this comment, done it in https://github.com/fbpic/fbpic/pull/582/commits/81dcec49d2bf007f06a0ab12115b4b6c307ebe04.